templates/test/ci/prow-machine-pool/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ namespace: default
 resources:
   - ../../../flavors/machinepool-windows
   - ../prow/cni-resource-set.yaml
+  - ../../../addons/windows/csi-proxy/csi-proxy-resource-set.yaml
```

Comment:
while you're here, could you please also add

`  - ../../../addons/windows/containerd-logging/containerd-logger-resource-set.yaml` 

seems that I missed that one too in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2947/files 

see https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2947/files#r1050166465

Reply:
done